lib/SILGen/SILGenBridging.cpp

```diff
@@ -946,6 +937,7 @@ SILGenFunction::emitBlockToFunc(SILLocation loc,
     auto loc = RegularLocation::getAutoGeneratedLocation();
     buildBlockToFuncThunkBody(thunkSGF, loc, blockType, funcType,
                               loweredBlockTy, loweredFuncTyWithoutNoEscape);
+    SGM.emitLazyConformancesForFunction(thunk);
```

Comment:
There are a lot of these `emitLazyConformancesForFunction` calls scattered around, and it seems easy to forget it when we add the next function that creates a new SIL function body. Could we put a NDEBUG-only check in the `SILGenFunction` constructor to ensure that `emitLazyConformancesForFunction` was called on that function? Or go really wild and call `emitLazyConformancesForFunction` from the destructor?